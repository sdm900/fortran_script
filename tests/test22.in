x=1
if (x==2) then
a(1)=1
else
b(1)=1
endif
print b(1)
print a(1)

