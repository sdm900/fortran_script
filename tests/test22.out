     1.00000 
 8 Unknown text: a(1)
